projects/plugins/crm/js/ZeroBSCRM.admin.global.js

```diff
@@ -2325,10 +2329,10 @@ function zeroBSCRMJS_bindGlobalContactFuncs() {
 				//text: "Are you sure you want to delete these?",
 				type: '',
 				showCancelButton: true,
-				confirmButtonColor: '#3085d6',
-				cancelButtonColor: '#d33',
+				confirmButtonColor: '#000',
+				cancelButtonColor: '#fff',
+				cancelButtonText: '<span style="color: #000">' + zeroBSCRMJS_globViewLang( 'cancel' ) + '</span>',
```

Comment:
😆

Reply:
It is the only way to to it using Sweet Alert without overriding all styles and messing around with custom js. 😢